src/web/routes/application/steps/decision/get-decision-status.test.js

```diff
@@ -127,3 +126,20 @@ test(`getDecisionStatus() should return ${PENDING} if address mismatches`, (t) =
   })
   t.end()
 })
+
+test(`getDecisionStatus() should return ${PENDING} if email and/or phone mismatches`, (t) => {
+  const mismatchingAddressResults = [
+    { ...ADDRESS_MATCHED_RESULT, mobilePhoneMatch: 'not_matched', emailAddressMatch: 'matched' },
+    { ...ADDRESS_MATCHED_RESULT, mobilePhoneMatch: 'not_matched', emailAddressMatch: 'not_matched' },
+    { ...ADDRESS_MATCHED_RESULT, mobilePhoneMatch: 'not_matched', emailAddressMatch: 'not_set' },
+    { ...ADDRESS_MATCHED_RESULT, mobilePhoneMatch: 'not_set', emailAddressMatch: 'matched' },
+    { ...ADDRESS_MATCHED_RESULT, mobilePhoneMatch: 'not_set', emailAddressMatch: 'not_matched' },
+    { ...ADDRESS_MATCHED_RESULT, mobilePhoneMatch: 'matched', emailAddressMatch: 'not_set' },
+    { ...ADDRESS_MATCHED_RESULT, mobilePhoneMatch: 'matched', emailAddressMatch: 'not_matched' }
```

Comment:
Can we add a couple with 'not_held' please?

Reply:
Done.